bool do_measurement
float32 distance
float32 driving_speed
float32 measure_speed
float32 measure_time
float32 acceleration
float32 deacceleration
float32 marge

